examples/rosa-classic-public-with-idp-machine-pools/main.tf

```diff
@@ -19,14 +18,33 @@ module "rosa" {
       "replicas" : 3
     }
   }
-  idp = {
-    "1" : {
-      "name" : "idp1",
-      "gitlab" = {
-        "client_id"     = "2189835314857134"
-        "client_secret" = "DG66575SKFGMdDFDSDSGTFSB47634735VDSFFDV"
-        "url"           = "https://gitlab.com"
-      }
-    }
-  }
+}
+
+module "gitlab_idp" {
+  source = "../../modules/idp"
+
+  cluster_id               = module.rosa.cluster_id
+  name                     = "gitlab-idp"
+  idp_type                 = "gitlab"
+  gitlab_idp_client_id     = random_password.client_id.result
```

Comment:
These (client id and secret) are not random numbers

Reply:
I'm using random number just to have something that can be executed by terraform apply. Otherwise we won't be able to run this for tests

Reply:
Added comment for this, to make it more clear